djmoney/_compat.py

```diff
@@ -1,33 +1,10 @@
-# -*- coding: utf-8 -*-
 # flake8: noqa
 
-
-try:
-    from django.utils.encoding import smart_unicode
-except ImportError:
-    # Python 3
-    from django.utils.encoding import smart_text as smart_unicode
-
-try:
-    string_types = (basestring,)
-    text_type = unicode
-except NameError:
-    string_types = (str, bytes)
-    text_type = str
-
-try:
-    # Python 2
-    reload_module = reload
-except NameError:
-    # Python 3.4+
-    from importlib import reload as reload_module
-
-try:
-    from urllib2 import urlopen
-    from urlparse import urlparse, parse_qsl, urlunparse
-except ImportError:
-    from urllib.request import urlopen
-    from urllib.parse import urlparse, parse_qsl, urlunparse
+# Formerly Python 2.7 diverted, so stored here
+string_types = (str, bytes)
```

Comment:
Do we need it anymore? It seems like we can inline all this code since there is no extra difference in Python versions and `_compat` is not that needed ... What do you think?

Reply:
This was me being sloppy :) But also kind of wanting to keep `_compat` because of the concept: Isolating this compatibility in places where it can easily be removed.

Not removing it despite of that confirmed maybe the extreme stubborn laziness of a programmer.. "oh a corner, let me cut that" :D

Reply:
:)))